ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/cli/CliSpec.scala

```diff
@@ -113,6 +113,9 @@ class CliSpec extends WordSpec with Matchers {
       checkOption(Array(s"--sql-backend-jdbcurl", jdbcUrl), _.copy(jdbcUrl = Some(jdbcUrl)))
     }
 
+    "parse the eager package loading flag when given" in {
```

Comment:
This text should rather read `"set the eager package loading flag when the respective command line flag is set"`, I believe.

Reply:
i'd rather be consistent with other flags, see above